examples/java/BUILD

```diff
@@ -9,7 +9,7 @@ load("//base:distro.bzl", "DISTRO_SUFFIXES")
 [java_image(
     name = "hello" + distro_suffix,
     srcs = ["HelloJava.java"],
-    base = "//java:java11" + distro_suffix,
+    base = "//java:java11_nonroot" + distro_suffix,
     main_class = "examples.HelloJava",
```

Comment:
It's worth testing both `root` and `nonroot`? (These are test examples that the test framework actually runs.)